test/libyul/EWasmTranslationTest.cpp

```diff
@@ -131,7 +131,7 @@ string EWasmTranslationTest::interpret()
 {
 	InterpreterState state;
 	state.maxTraceSize = 10000;
-	state.maxSteps = 10000;
+	state.maxSteps = 100000;
```

Comment:
Is this still needed?

Reply:
Yes. For mod, addmod and mulmod test cases. These test cases need more then 1000 steps to run.